### ISSUE TITLE
Home Assistant: Add language translation dict for states that come back from HA

### ISSUE THREAD

**Issue by JarvyJ** [JarvyJ/HomeIntent#275]
Yeah, this is kinda interesting. Since it affects just Home Assistant, I think I might add a language dictionary that gets auto-loaded based on language settings and provided back in the base constructor for use. English words that need to be translated from Home Assistant could all be defined there with their translations and the code could reference. I'll create a new issue and play around with it a bit to see how it would work.

_Originally posted by @JarvyJ in https://github.com/JarvyJ/HomeIntent/issues/232#issuecomment-1006215905_

**Comment by JarvyJ** [comment]
Hello @oerkel47! I think I have a solution for the HA API returning states/attributes in English. Since it only affects the Home Assistant component, I've created an [`api_translations`](https://github.com/JarvyJ/HomeIntent/tree/b3e9611c3f9a2eacfac013b4bf1ae81ef1752754/home_intent/components/home_assistant/api_translations) folder with the languages' translations that come back from the HA API (in a new branch - #279)

The idea being that any English response that comes back from the API indicating a state (like `on` or `off`) is defined in the `api_translations/[language].json` file. This gets autoloaded by the Home Assistant component and is setup in the `HomeAssistant` component.

In all of the domain-specific controller and their language files it can be accessed with `self.ha.api_translation`. So in `switch/de.py`, you could use `self.ha.api_translations["on"]` and it would return `an`.

I kinda like the idea of having all the responses from the API translated in the same place. If you want to try out the branch and let me know how it goes that would be great! I tried it out a bit and I _hope_ it'll be useful overall.

**Comment by oerkel47** [comment]
Thanks @JarvyJ, I will try it out soon.

**Comment by oerkel47** [comment]
Wouldn't it be a good idea to directly translate the translatable entries inside the response dict  inside the base class? Would take the work from the translators while also keeping the language specific code tidy. 

`response = self._toggle_light(light)
response["state"] = self.ha.api_translation[response["state"]]`

This works, but looks ugly.

**Comment by JarvyJ** [comment]
Oh, I like that idea! It'll make it a lot nicer for translators. I'll see if I can get it worked in today along with a workaround for #274

**Comment by JarvyJ** [comment]
The branch has been updated!

So I ended up doing the translation work in the Home Assistant [api](https://github.com/JarvyJ/HomeIntent/pull/279/files#diff-0c00a4127ccfa15349794d85935239b3d318127bd5851c4913270bb776e4c29f) code. This way any `get_entity` call that has a `state` attribute will get translated with the json file.

I've also made the `get_entity` to take in the response from `call_service` and check if it's an empty list, perform the delay, and then return. This should solve the issue in #274.

I'll do a bit more testing on my end, and if it seems good for you, we can merge and go from there!
